native/packages/react-native-bpk-component-image/stories.js

```diff
@@ -0,0 +1,151 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { Component } from 'react';
+import { View, Image, StyleSheet } from 'react-native';
+import BpkButton from 'react-native-bpk-component-button';
+import { spacingBase, spacingLg } from 'bpk-tokens/tokens/base.react.native';
+import { storiesOf } from '@storybook/react-native';
+
+import CenterDecorator from '../../storybook/CenterDecorator';
+import BpkImage from './index';
+
+const styles = StyleSheet.create({
+  button: {
+    marginBottom: spacingBase,
+  },
+  image: {
+    width: '100%',
+    height: 10 * spacingLg,
+  },
+});
+
+class BpkImageLoadingStory extends Component<Props, State> {
+  static propTypes = {};
+
+  static defaultProps = {};
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      loaded: false,
+    };
+  }
+
+  onLoadToggle = () => {
+    this.setState(prevState => ({ loaded: !prevState.loaded }));
+  };
+
+  render() {
+    return (
+      <View>
+        <BpkButton
+          style={styles.button}
+          title={
+            this.state.loaded ? 'Reload story to try again' : 'Mark as loaded'
+          }
+          onPress={this.onLoadToggle}
+          disabled={this.state.loaded}
+        />
+        <BpkImage
+          loaded={this.state.loaded}
+          alt="test"
+          style={styles.image}
+          {...this.props}
+        />
+      </View>
+    );
+  }
+}
+
+storiesOf('react-native-bpk-component-image', module)
+  .addDecorator(CenterDecorator)
+  .add('default', () => (
+    <BpkImage
+      alt="test"
+      source={{
+        uri:
+          'https://content.skyscnr.com/96508dbac15a2895b0147dc7e7f9ad30/canadian-rockies-canada.jpg',
+      }}
+      style={styles.image}
+    />
+  ))
+  .add('default png', () => (
+    <BpkImage
+      alt="test"
+      source={{
+        uri:
+          'https://upload.wikimedia.org/wikipedia/commons/thumb/3/36/Two_red_dice_01.svg/2000px-Two_red_dice_01.svg.png',
+      }}
+      style={styles.image}
+    />
+  ))
+  .add('no border radius', () => (
+    <BpkImage
+      borderRadius={false}
+      alt="test"
+      source={{
+        uri:
+          'https://content.skyscnr.com/200946ddb82b7c026e6e186a7037b1f8/machu-picchu.jpg',
+      }}
+      style={styles.image}
+    />
+  ))
+  .add('loading', () => (
+    <BpkImageLoadingStory
+      source={{
+        uri:
+          'https://content.skyscnr.com/8a8ac17b591b61e6fe5d8f63414561cd/amsterdam-the-netherlands.jpg',
+      }}
+    />
+  ))
+  .add('loading png', () => (
+    <BpkImageLoadingStory
+      source={{
+        uri:
+          'https://upload.wikimedia.org/wikipedia/commons/thumb/3/36/Two_red_dice_01.svg/2000px-Two_red_dice_01.svg.png',
+      }}
+    />
+  ))
+
+  .add('control', () => (
```

Comment:
We can probably replace this one now. A performance testing story would be good, I think. Bang 100 images in a ScrollView and check scroll performance vs doing the same with regular images.